Examples-Web/HoverButton/Version.txt

```diff
@@ -0,0 +1,3 @@
+TouchFree Tooling for Web Version: 1.1.1
+TouchFree Tooling API Version: 1.2.0
+Release Reference: deb6ae91
```

Comment:
Not seen this before in our demos, where's this getting read?

Reply:
It's present in the other button's folder (https://github.com/ultraleap/TouchFree-Tooling-Examples/tree/develop/Examples-Web/AnimatedProgressButtons). I copied this Version.txt from the Tooling release folder

Reply:
It isn't read anywhere - more so you know the version of the build tooling file

Reply:
Ah yeah, missed that one. Seems like a good thing :+1: